tests/core/editable/domfix3.js

```diff
@@ -0,0 +1,30 @@
+/* bender-tags: editor,edge */
+
+bender.test( {
+	// This test is as dirty as hack for Edge itself. To detect if reselection in domFix is not
+	// happening for the first focus, we stub selection's selectRanges to detect any
+	// selection after focus (#504).
+	'test first focus in Edge': function() {
```

Comment:
This test checks some internals and doesn't check the result (correctly placed selection) for this fix. I assume creating the test which will focus the editor and then check selection position (and maybe scroll selection) was not plausible? Can you shed some light on this @Comandeer?

Reply:
IIRC I just couldn't find any sensible way to focus the editor and not create selection at the beginning of the content at the same time. It seems to behave in such way only during user interaction. Due to that it's impossible to test selection position after first focus.

Reply:
That was my assumption. If such behaviour is caused by user interaction only we may go with what you proposed (in such situations we may also simply skip unit test, but since it's already done let's keep it) 👍